OpenRA.Mods.Common/Traits/AutoCarryall.cs

```diff
@@ -9,38 +9,67 @@
  */
 #endregion
 
+using System.Collections.Generic;
 using System.Linq;
 using OpenRA.Activities;
 using OpenRA.Mods.Common.Activities;
+using OpenRA.Support;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[Desc("Automatically transports harvesters with the Carryable trait between resource fields and refineries.")]
+	[Desc("Automatically transports harvesters with the AutoCarryable and CarryableHarvester between resource fields and refineries.")]
 	public class AutoCarryallInfo : CarryallInfo
 	{
+		[ConsumedConditionReference]
+		[Desc("Boolean expression defining the condition under which the auto carry behavior is enabled. Enabled at default.")]
```

Comment:
Shouldn't this be "by default" rather then "at default"?

Reply:
fixed